06-vendoring/README.md

```diff
@@ -0,0 +1,63 @@
+# Vendoring Packages: FastAPI + Jinja2 Example
+
+*Note: You must have Python Packages enabled on your account for built-in packages to work. Request Access to our Closed Beta using [This Form](https://forms.gle/FcjjhV3YtPyjRPaL8)*
+
+This is an example of a Python Worker that uses a built-in package (FastAPI) with a vendored package (Jinja2).
+
+## Adding Packages
+
+Built-in packages can be selected from [this list](https://developers.cloudflare.com/workers/languages/python/packages/#supported-packages) and added to your `requirements.txt` file. These can be used with no other explicit install step.
+
+Vendored packages are added to your source files and need to be installed in a special manner. The Python Workers team plans to make this process automatic in the future, but for now, manual steps need to be taken.
+
+### Vendoring Packages
+
+First, install Python3.12 and pip for Python 3.12.
+
+*Currently, other versions of Python will not work - use 3.12!*
+
+Then create a virtual environment and activate it from your shell:
+```console
+python3.12 -m venv .venv
+source .venv/bin/activate
```

Comment:
I assume these instructions are intended to be mac/linux specific? Maybe we can later add an OS picker that shows windows instructions... Though none of this is really tested on windows at all.

Reply:
Maybe we should change these instructions to use `uv`? it makes installing a specific python version really easy (https://docs.astral.sh/uv/guides/install-python/#installing-a-specific-version) so should make it much easier for people to get this going.

Reply:
No objection.